tedana/metrics/dependence.py

```diff
@@ -168,31 +168,31 @@ def calculate_f_maps(data_cat, Z_maps, mixing, adaptive_mask, tes, f_max=500):
 
             # S0 Model
             # (S,) model coefficient map
-            coeffs_S0 = (comp_betas[:j_echo] * X1[:j_echo, :]).sum(axis=0) / (
-                X1[:j_echo, :] ** 2
+            coeffs_s0 = (comp_betas[:j_echo] * x1[:j_echo, :]).sum(axis=0) / (
+                x1[:j_echo, :] ** 2
             ).sum(axis=0)
-            pred_S0 = X1[:j_echo, :] * np.tile(coeffs_S0, (j_echo, 1))
-            SSE_S0 = (comp_betas[:j_echo] - pred_S0) ** 2
-            SSE_S0 = SSE_S0.sum(axis=0)  # (S,) prediction error map
-            F_S0 = (alpha - SSE_S0) * (j_echo - 1) / (SSE_S0)
-            F_S0[F_S0 > f_max] = f_max
-            F_S0_maps[mask_idx, i_comp] = F_S0[mask_idx]
+            pred_s0 = x1[:j_echo, :] * np.tile(coeffs_s0, (j_echo, 1))
+            sse_s0 = (comp_betas[:j_echo] - pred_s0) ** 2
```

Comment:
This is the same issue I mentioned in collect.py, but it's all over this file. `SSE` is a capitalized acronym.